File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        maxcount=0
        count=0
        n=len(nums)
        for i in range(0,len(nums)):
            if(nums[i]==1):
                count+=1
            else:
                maxcount=max(count,maxcount)
                count=0
        return max(maxcount,count)
        """
        :type nums: List[int]
        :rtype: int
        """
        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count=0;
        int maxcount1=0;
        int maxcount2=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==1)
            count++;
            else if(nums[i]==0){
                maxcount1=max(count,maxcount1);
                count=0;
            }
            // if(maxcount<count);
            // maxcount=count;
        }
        
             maxcount2=count;
             int ans=max(maxcount1,maxcount2);
        
        return ans;
        
    }
};